cmd/sol_refresher/cmd/coin.go

```diff
@@ -201,6 +213,16 @@ var coinCmd = &cobra.Command{
 			updates = append(updates, sql)
 		}
 
+		for position, txSig := range dammV2Inits {
+			fmt.Println("Refreshing Damm V2 init tx:\t", txSig)
```

Comment:
Is all of the output here using prints instead of a logger because it's a command line tool?